### ISSUE TITLE
v2 Upgrade commit detail

### ISSUE THREAD

**Issue by jasonpnnl** [microsoft/azurechat#350]
@thivy Would you be able to provide a repo with a more detailed commit history for the v2 upgrade? In the case where v1 was in use and heavily modified, it would simplify the upgrade process to be able to have more commit history and apply changes that way rather than all of the v2 changes being applied in the single update commit.

**Comment by ianwdunlop** [comment]
I agree. More documentation would be very helpful. There seems to be more env vars than there are details of the services that use them and how to set them up.

**Comment by olafwrieden** [comment]
Hi @jasonpnnl there is some detail in the migration readme (in /docs) on the topic of migration. I'll go ahead and close this issue... going forward the migration is where major changes to the application will be documented.
